react-api-client/src/camera/useCameraSettings.ts

```diff
@@ -0,0 +1,63 @@
+import { useMutation, useQueryClient } from 'react-query'
+
+import { createCameraSettings } from '@opentrons/api-client'
+
+import { useHost } from '../api'
+
+import type { AxiosError } from 'axios'
+import type {
+  UseMutateFunction,
+  UseMutationOptions,
+  UseMutationResult,
+} from 'react-query'
+import type {
+  CameraResponse,
+  CameraSettings,
+  ErrorResponse,
+} from '@opentrons/api-client'
+import type { CameraId } from '@opentrons/shared-data'
+
+export type UseUpdateCameraMutationResult = UseMutationResult<
+  CameraResponse,
```

Comment:
```suggestion
  CameraSettingsResponse,
```
`CameraResponse` points to the usage settings as the underlying data. We'll need to swap out all usages here.